themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```
@@ -643,6 +643,10 @@ single-logout-service-url=Single Logout Service URL
 saml.single-logout-service-url.tooltip=The Url that must be used to send logout requests.
 nameid-policy-format=NameID Policy Format
 nameid-policy-format.tooltip=Specifies the URI reference corresponding to a name identifier format. Defaults to urn:oasis:names:tc:SAML:2.0:nameid-format:persistent.
+saml.principal-type=Principal
+saml.principal-type.tooltip=Specifies how Keycloak should identify and track external users. Default is using Subject NameID, though it is recommended to set up identifying attribute which is more reliable.
```

No opinions please, just pure description.

```suggestion
saml.principal-type.tooltip=Way to identify and track external users from the assertion. Default is using Subject NameID, alternatively you can set up identifying attribute.
```